### PR TITLE
Add directory separator when replacing localDir with bucket name

### PR DESCRIPTION
**Pull request by nibynool**
I missed one line in my previous fix for #41.

Prior to merging this I have asked @gs11 to verify that the fix works.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -232,7 +232,7 @@ class ServerlessS3Sync {
             ...contentTypeObject,
             ...file.params,
             ...{
-              CopySource: file.name.replace(localDir, `${s.bucketName}${bucketPrefix == '' ? '' : bucketPrefix}/`),
+              CopySource: file.name.replace(localDir + path.sep, `${s.bucketName}${bucketPrefix == '' ? '' : bucketPrefix}/`),
```
Comment (nibynool) [inline index.js]: I have left the trailing slash after the bucket prefix to conform with AWS expectations.  To allow for this I added the path.sep (operating system specific) on the original file name

**Comment by nibynool** [comment]
@gs11 has confirm this works in https://github.com/k1LoW/serverless-s3-sync/issues/41#issuecomment-579726296

**Comment by k1LoW** [comment]
@nibynool Thank you !!!! Sorry for late reply..

**Comment by k1LoW** [comment]
Released as v1.10.3.

**Pull request merged**